ingestion/src/metadata/ingestion/source/database/redshift/queries.py

```diff
@@ -203,6 +203,11 @@
             """
 )
 
+REDSHIFT_EXTERNAL_TABLE_LOCATION = """
+    SELECT location 
+    FROM svv_external_tables 
+    WHERE schemaname = :schema AND tablename = :table
```

Comment:
we prefer using `{schema}`  and then `string.format(schema=schema_name)` instead of using sql binding.